codec-http2/src/main/java/io/netty/handler/codec/http2/Http2SecurityUtil.java

```diff
@@ -34,26 +34,48 @@
      * href="https://wiki.mozilla.org/Security/Server_Side_TLS#Non-Backward_Compatible_Ciphersuite">Mozilla Cipher
      * Suites</a> in accordance with the <a
      * href="https://tools.ietf.org/html/draft-ietf-httpbis-http2-16#section-9.2.2">HTTP/2 Specification</a>.
+     * 
+     * On OpenJDK, the ciphers start with TLS_, while on IBM J9 they are named SSL_.
```

Comment:
Well i consider Oracle JDK to be OpenJDK, but I happily will adjust to your preferred wording.
But the TLS_ names work correctly on Azul Zulu, so we should add that, or say OpenJDK based JVMs?